src/orchestration/Orchestration.ts

```diff
@@ -148,6 +149,7 @@ export type Config = {
     batchLimit: number;
     clientBuilder?: ClientBuilder;
     cookieAttributes: CookieAttributes;
+    sessionAttributes?: { [k: string]: string | number | boolean };
```

Comment:
nit: Make this required and make it default to an empty object.
```suggestion
    sessionAttributes: { [k: string]: string | number | boolean };
```